src/qml/FeatureForm.qml

```diff
@@ -647,7 +647,7 @@ Page {
         bgcolor: Theme.darkGray
 
         iconSource: Theme.getThemeIcon( "ic_check_white_48dp" )
-        opacity: featureFormList.model.constraintsHardValid ? 1.0 : 0.3
+        opacity: typeof featureFormList !== "undefined" ? featureFormList.model.constraintsHardValid ? 1.0 : 0.3 : 1.0
```

Comment:
I think it is safer to either check for exact like `instanceof(featureFormList, FormListModel)`, or be super relaxed what it should not be. So far I have fixed this kind of warnings like my proposal, since it might be `null` in some cases.
```suggestion
        opacity: featureFormList ? featureFormList.model.constraintsHardValid ? 1.0 : 0.3 : 1.0
```

Reply:
@suricactus , I tried this first, but it still gave me an error on launch.

Reply:
Just tested. You are right. Interesting though... Go ahead!

Reply:
@suricactus , I'm as perplexed as you here. But typeof works so... going ahead! :)